client/reader/stream/index.jsx

```diff
@@ -83,6 +92,20 @@ class ReaderStream extends Component {
 		forcePlaceholders: false,
 	};
 
+	constructor( props ) {
+		super( props );
+		this.state = {
+			selectedTab: 'posts',
+		};
+	}
```

Comment:
No need to use `constructor()` if we just want to declare default component state:

```suggestion

	state = {
		selectedTab: 'posts',
	};
```